DevoxxClientMobile/build.gradle

```diff
@@ -3,7 +3,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'org.javafxports:jfxmobile-plugin:1.3.17'
+        classpath 'org.javafxports:jfxmobile-plugin:1.3.18'
```

Comment:
With this, now you can remove `javafxportsVersion = "..."`.

Reply:
IMO, its always better to have the `javafxportsVersion` defined explicitly :)

Reply:
That could lead to potential problems, if we later on release a new version, and someone only increases the plugin version. Two versions to update are worse that just one :)